src/main/scala/nl.knaw.dans.easy.download/EasyDownloadServlet.scala

```diff
@@ -56,8 +57,21 @@ class EasyDownloadServlet(app: EasyDownloadApp) extends ScalatraServlet
     }
   }
 
-  private def getUUID = Try {
-    UUID.fromString(params("uuid"))
+  private def getUUID = {
+    uuidAfterHyphenCheck(params("uuid")).toUUID match {
+      case Right(uuid) => Success(uuid)
+      case Left(error) => Failure(error)
+    }
+  }
+
+  private def uuidAfterHyphenCheck(uuid: String): String =  {
```

Comment:
* Name of this function could be better. The word `Check` in a function name leads me to think that it performs some check (and can therefore cause a failure). This function does no such thing. It just normalizes the input into a standard form. How about: `correctHyphenation(uuid)`?
* If hyphens are without meaning in ARK then a UUID with the hyphens in the wrong places should also succeed (as long as it has 32 hex digits, of course).

Reply:
I changed the method as you suggested.

But I disagree that we should accept `uuids` where the hyphens are in wrong places.  In my opinion the fact that hyphens in `ARK standard` are not significant means that we should accept `uuids` without hyphens, but not if the user has chosen to use the hyphens and puts them in wrong positions.  For the user those hyphens in that case are significant :) and we would give a wrong message by relocating the hyphens into other places.

Reply:
OK, decided during dev meeting: we will accept no hyphens or hyphens in the correct places. This must be documented in `docs/api/api.yml`. Be sure to check the OpenApi synax in the Swagger editor: https://editor.swagger.io/.